src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,106 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+
+        checkCreateNewCarMethod(carService);
+
+        getCarById(carService, 1L);
+
+        getAllCars(carService);
+
+        updateCar(carService);
+
+        deleteCarById(carService, 4L);
+
+        getAllCarsByDriver(carService, 3L);
+
+        addNewDriverToCar(carService);
+
+        removeDriverFromCar(carService);
```

Comment:
maybe it'll be better to remove these empty lines?
```suggestion
        checkCreateNewCarMethod(carService);
        getCarById(carService, 1L);
        getAllCars(carService);
        updateCar(carService);
        deleteCarById(carService, 4L);
        getAllCarsByDriver(carService, 3L);
        addNewDriverToCar(carService);
        removeDriverFromCar(carService);
```